test/text/vtt_text_parser_unit.js

```diff
@@ -1100,8 +1100,29 @@ describe('VttTextParser', () => {
           {
             startTime: 110,
             endTime: 120,
-            payload: '<c.lime>less or more < > in text</c>',
```

Comment:
The removal of `< >` from this text potentially changes the meaning of the text and is not good for a captioning system.

Can these be replaced by html entities (like `&lt;` and `&gt;`)? I guess it would depend on how this ends up being rendered eventually.

Reply:
Yes, I think we should not remove these characters.

Reply:
The cost of replacing these characters will cost a little performance I will do some tests and see what the best option is.